module/Database/src/Service/Member.php

```diff
@@ -618,6 +618,12 @@ public function canRemove(MemberModel $member): bool
      */
     public function remove(MemberModel $member): void
```

Comment:
Out of scope for this PR, but we really should wrap all of this in a transaction.